userspace/libsinsp/CMakeLists.txt

```diff
@@ -311,3 +316,43 @@ if (BUILD_LIBSINSP_EXAMPLES)
 	add_subdirectory(examples)
 endif()
 
+# Build our pkg-config "Libs:" flags. For now, loop over SINSP_LIBRARIES. If
+# we ever start using pkg_search_module or pkg_check_modules in cmake/modules
+# we could add each module to our "Requires:" line instead. We might need to
+# expand this to use some of the techniques in
+# https://github.com/curl/curl/blob/curl-7_84_0/CMakeLists.txt#L1539
+set(SINSP_PKG_CONFIG_LIBS)
+set(SINSP_PKG_CONFIG_LIBDIRS)
+foreach(sinsp_lib ${SINSP_LIBRARIES})
+	if(${sinsp_lib} MATCHES "^-")
+		# We have a flag. Pass it through unchanged.
+		list(APPEND SINSP_PKG_CONFIG_LIBS ${sinsp_lib})
+	elseif(${sinsp_lib} MATCHES "/" OR ${sinsp_lib} MATCHES "^-")
```

Comment:
Aren't we repeating `${sinsp_lib} MATCHES "^-"` here?

Reply:
It was a copy+paste error. Thanks for catching that.